java/client/src/main/java/glide/api/commands/ConnectionManagementCommands.java

```diff
@@ -14,17 +14,16 @@ public interface ConnectionManagementCommands {
      * Ping the Redis server.
      *
      * @see <a href="https://redis.io/commands/ping/">redis.io</a> for details.
-     * @return Response from Redis containing a <code>String</code> with "PONG".
+     * @return <code>String</code> with "PONG".
```

Comment:
Time to polish the doc!
```suggestion
     * @return <code>String</code> with <code>PONG</code>.
```
or
```suggestion
     * @return <code>String</code> with a server response.
```
(update all docs if you start)

Reply:
I can do the first... ;)